docker/lite/build.sh

```diff
@@ -3,15 +3,48 @@
 # This is the script to build the vitess/lite Docker image by extracting
 # the pre-built binaries from a vitess/base image.
 
+# Parse command line arguments.
+prompt_notice=true
+if [[ "$1" == "--prompt"* ]]; then
+  if [[ "$1" == "--prompt=false" ]]; then
+    prompt_notice=false
+  fi
+  shift
+fi
+
 flavor=$1
 
+tag=latest
 if [[ -n "$flavor" ]]; then
   base_image=vitess/base:$flavor
+  tag=$flavor
 else
   echo "Flavor not specified as first argument. Building default image."
   base_image=vitess/base
 fi
 
+# Abort if base image does not exist.
+docker inspect $base_image &>/dev/null
+if [[ $? -ne 0 ]]; then
+  echo "ERROR: Dependent image $base_image does not exist. Run 'make docker_base' to build it locally or 'docker pull $base_image' to fetch it from Docker Hub."
+  exit 1
+fi
+
+# Educate the user that they have to build or pull vitess/base themselves.
+if [[ "$prompt_notice" = true ]]; then
+  echo
```

Comment:
I've used `cat <<END` as mentioned in the example in the Google Bash Style Guide: https://google.github.io/styleguide/shell.xml?showone=Line_Length_and_Long_Strings#Line_Length_and_Long_Strings